test-browser/simpleContract.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+var fs = require('fs');
+
+module.exports = {
+  'Simple Contract': function (browser) {
```

Comment:
Isn't it better to use nodejs require for that?
Another solution (perhaps for the next iteration) would be to declare sources inside each test with a special delimiter and scan all test files when running `makeMockCompiler`.